### PR TITLE
linux/fuego-multinode: add readme.md

### PR DESCRIPTION
**Pull request by chase-qi**
Signed-off-by: Chase Qi <chase.qi@linaro.org>

### CODE REVIEW

**Comment by chase-qi** [comment]
Once we get this reviewed/merged, maybe, we should talk to Fuego developers to add the content to http://fuegotest.org/wiki/Using_Fuego_with_LAVA to make it easy to find.

**Comment by fboudra** [comment]
@chase-qi and get them to upgrade their container from Jessie to Stretch, which bring a more recent LAVA and also deprecate lava-tool in favor of lavacli.

File: automated/linux/fuego-multinode/README.MD [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    {% set docker_cpus = '2' %}
+    {% set docker_memory = '2048M' %}
+    {% set docker_devices = ["/dev/kvm:/dev/kvm"] %}
+    {% set docker_volumes = ["/var/lib/lava/dispatcher/tmp:/var/lib/lava/dispatcher/tmp", "/boot:/boot", "/lib/modules:/lib/modules"] %}
+    ```
+    
+    Refer to https://staging.validation.linaro.org/scheduler/device/staging-docker-03/devicedict
+
+    Notes: 
+    * At the time of writing this guide, fuego master branch, [v1.4.0](https://bitbucket.org/fuegotest/fuego/commits/8b16eda803d4627f66a0102eac48d00b6857c801), still depends on Jenkins for test run from command line. Jenkins is memory hungry, 1G isn't enough, 2G is recommended.
```
Comment (fboudra) [inline automated/linux/fuego-multinode/README.MD]: fwiw, it's inaccurate. Jenkins can run on 512M. Memory usage depends on the number of nodes connected and the number of concurrent build jobs, etc...

Comment (chase-qi) [reply automated/linux/fuego-multinode/README.MD]: That is right, adding new fuego nodes eats more. We started with 1G, but that is not enought, adding node hangs. It is recorded here https://projects.linaro.org/browse/LSS-531

Comment (fboudra) [reply automated/linux/fuego-multinode/README.MD]: So I believe this bit can be rewritten with the context, something like: Our testing of Fuego container showed at least 2G are required, defined in the docker device dictionary using docker_memory.

Comment (chase-qi) [reply automated/linux/fuego-multinode/README.MD]: Thanks for advice. Updated.

**Comment by chase-qi** [comment]
> @chase-qi and get them to upgrade their container from Jessie to Stretch, which bring a more recent LAVA and also deprecate lava-tool in favor of lavacli.

@sangorrin ^ may have more comments.

**Pull request merged**